compiler/src/dotty/tools/dotc/core/Types.scala

```diff
@@ -2802,7 +2794,10 @@ object Types {
     def syntheticParamName(n: Int) = tpnme.syntheticTypeParamName(n)
   }
 
-  abstract class MethodTypeCompanion extends TermLambdaCompanion[MethodType] {
+  abstract class MethodTypeCompanion extends TermLambdaCompanion[MethodType] { self =>
+
+    def isJava: Boolean = false
+    def isImplicit: Boolean = false
```

Comment:
I don't think these are needed either?

Reply:
They will become useful if we add more method types. I will remove it and add it back if it is needed in the future.